docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,357 @@
+# Getting Started
+This document is intended as a "getting started" guide. The audience are developers looking to modify or *publish* the packages to the Universe. This document is written in tutorial and walk-through format. The goal is to help you "get started". It does not go into great depth on some of the conceptual or inner details. This guide aims at making a user familiar with the concepts of what a Package is and what are the roles of Marathon and Universe in the package life cycle.
+
+
+## Prerequisites
+Before starting this guide, make sure you have met the following conditions.
+
+### Library dependencies
+* [DC/OS CLI](https://dcos.io/docs/latest/cli/install/) installed and configured.
+* [jq](https://stedolan.github.io/jq/download/) is installed in your environment.
+* python3 in your environment.
+* Docker is installed.
+
+### Access requirements
+* Access to a running [DC/OS](https://dcos.io/docs/latest/overview/what-is-dcos/).
+* The Universe Server needs to be built and run in a location accessible by the DC/OS Cluster. There is no other way to test your package otherwise.
```

Comment:
It would make sense to move the `required nomenclature` section before this. Also, explain `Universe Server` or other terminology being used.